test/check-verify

```diff
@@ -202,7 +202,7 @@ def main():
     if not revision:
         revision = subprocess.check_output([ "git", "rev-parse", "HEAD" ]).strip()
     if not context:
-        context = testinfra.DEFAULT_IMAGE
+        context = opts.image or testinfra.DEFAULT_IMAGE
```

Comment:
Can't we just do this directly when context is declared? If we're scanning github, it'll just be overwritten, but the default would be clearer.